sonic_psu/psu_base.py

```diff
@@ -26,10 +26,22 @@ def get_num_psus(self):
     @abc.abstractmethod
     def get_psu_status(self, index):
         """
-        Retrieves the oprational status of power supply unit (PSU) defined
+        Retrieves the operational status of power supply unit (PSU) defined
                 by index <index>
 
         :param index: An integer, index of the PSU of which to query status
         :return: Boolean, True if PSU is operating properly, False if PSU is faulty
```

Comment:
Could you also please add more detail to my description here to reflect what we have discussed as "faulty" (e.g., if the PSU is present but powered off, it is considered a fault, and should return false.

Reply:
More details has been added